### ISSUE TITLE
Auriculotemporal Nerve Injury

### ISSUE THREAD

**Issue by MickeySegal** [obophenotype/human-phenotype-ontology#10151]
**Preferred term label:**
Auriculotemporal Nerve Injury

**Synonyms**

**Definition (free text, please give PubMed ID)**
https://uts.nlm.nih.gov/uts/umls/concept/C5200768

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
Frey Syndrome

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
@MickeySegal 
I think that in the vast majority of cases, what one observes in Frey syndrome is this :" sweating and flushing in the preauricular area in response to mastication or a salivary stimulus", and what one infers is that there was an injury to the Auriculotemporal nerve. Inferences are not usually good HPO terms. I am wondering if this is truly a primary phenotypic observation or if we can provide a definition that specifies this?

**Comment by MickeySegal** [comment]
We have changed our finding in the same direction of observable facts, using the explanatory terms: Sweating, flushing, warmth in the auriculotemporal branch of the mandibular branch of the trigeminal nerve

**Comment by MickeySegal** [comment]
Typo: label: **Wweating** and flushing in the preauricular area in response to mastication

**Comment by pnrobinson** [comment]
@MickeySegal 
fixed, thanks!
